### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by pwilson77**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
```
Comment (inhaq) [inline index.html]: Consider adding a `lang` attribute to the `html` start tag to declare the language of this document.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+		</button>
+	</div>
+	<div class="form-container">
+		<div class="header">
+			<div class="logo">
+				<a href="#"><img src="logo-intuit-black.jpg" alt="logo-intuit-black"></a>
+			</div>
+			<div class="navbar">
+				
+				<img src="mint_2018_icon.png"> Mint
```
Comment (inhaq) [inline index.html]: An `img` element must have an `alt` attribute.

**Review by inhaq — APPROVED** [review]
### Job well done 🏆🥇
Congrats on completing this project. Before going on to submit a project completion form, I want you to make a few minor updates. No need to submit for review again.
Put all images in a folder and off course update the paths of images as well in respective files. Then do these minor changes as added in other comments below. 
Suggestions: Don't use `.jpg` images where you need transparency, you'll need to use `.png` format images.
You guys have done great, enjoy and good luck for the next one 👍🏻🎉

File: index.html (hunk 1 above)
Comment (truetechcode) [reply index.html]: Thanks for the review.

**Pull request merged**